vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/client/view/common/BasicBrowserToolStrip.java

```diff
@@ -80,7 +80,7 @@ public void onClick(ClickEvent event) {
                 if (!pathItem.getValueAsString().equals(DataManagerConstants.ROOT)) {
                     String newPath = pathItem.getValueAsString();
                     BrowserUtil.loadData(modal, grid, toolStrip,
-                            newPath.substring(0, newPath.lastIndexOf("/")), false);
+                            newPath.substring(0, newPath.lastIndexOf("/")), true);
```

Comment:
I think this refreshes automatically when going 1 folder up.
I am not sure we want this, as this could be long.